components/Admin/OrderContent.js

```diff
@@ -0,0 +1,115 @@
+import { ExpandMore } from '@mui/icons-material';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Divider,
+  Grid,
+  Stack,
+  Box,
+} from '@mui/material';
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+import { OrderPropType } from '../../globals';
+const OrderContent = ({ order, isMobile }) => {
+  const date = new Date(order.date);
+  const formattedTime = `${date.getHours().toString().padStart(2, '0')}:${date
```

Comment:
`date.getHours().toString().padStart(2, '0')`

seeing this repeated multiple times, maybe can extract into a helper function?